.gitignore

```diff
@@ -18,6 +18,7 @@ notebooks/01-use-cases/finance/risk-management/credit-risk/xva/data-prev/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/xva/data/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/xva/what-if/*.csv
 notebooks/01-use-cases/finance/risk-management/credit-risk/ifrs9/lending-club-data/*.csv
```

Comment:
@printhellohetal Was taking a look at our `.gitignore` to ignore the data files that are created from the Credit Card Analytics data processing notebook, and saw these lines that are no longer registering due to the deprecated filepaths. Will be fixing this, and this a symptom of an overarching issue indicated here: https://activeviam.atlassian.net/browse/AC-85